File: examples/java/simple/Example1.java
// Example 1: SET a value.
public class Example1 {
    public static void main(String[] args) throws Exception {
        MdisClient client = MdisClient.connect("127.0.0.1", 6411);
        String resp = client.set("token", "123456");
        System.out.println("response: " + resp);
    }
}

File: examples/php/simple/example_1.php
<?php

// Example 1: SET a value.
require_once __DIR__ . "/src/MdisClient.php";

$client = MdisClient::connect("127.0.0.1", 6411);
$resp = $client->set("token", "123456");
echo "response: {$resp}\n";
